packages/twenty-front/src/modules/object-record/utils/generateEmptyFieldValueForClear.ts

```diff
@@ -0,0 +1,73 @@
+import { FieldType } from '@/object-record/record-field/types/FieldType';
+import { FieldMetadataType } from '~/generated/graphql';
+import { capitalize } from '~/utils/string/capitalize';
+
+export const generateEmptyFieldValueForClear = (
```

Comment:
Maybe we could reuse `generateEmptyFieldValue` instead of creating another function? Perhaps adding a "isForClear" parameter to `generateEmptyFieldValue`?

Reply:
I'm waiting for https://github.com/twentyhq/twenty/pull/4378 to be merged to properly refactor this.